### DISCUSSION TITLE
Factors affecting the accuracy of lattice thermal conductivity

### DISCUSSION THREAD

**Discussion by apandit2021** [ttadano/alamode#32] [Q&A] [answered]
Dear Dr. Tadano,

What are the main factors (tags) that affect the accuracy of the lattice thermal conductivity (k_l) within ALAMODE implementation? Could you please see if the input tags below for RTA mode look reasonable?

&general
  PREFIX = 551_RTA_180by180
  MODE = RTA
  FCSXML = 551_cubic.xml
  NKD = 2
  KD = ........
  EMIN = 0; EMAX = 1000; DELTA_E = 1

/

&cell
50.00 # factor in Bohr unit
0.1753993041	.............          ...........
..................................................................
/

&kpoint
  2  # KPMODE = 2: uniform mesh mode
180 180 1
/

The obtained k_l values using the  ALAMODE code are way off than the values in the literature(s). As an example: the calculated k_l value (0.13 W/mK) at 300 K here is much less than the literature value (~ 4 W/mK) for a 2D material, where both the calculations are performed using the 3rd order Force constants using the RTA approach.
Thank you for your help.

Best,
Abhiyan

**Comment by ttadano** [comment, marked as answer]
In ALAMODE, thermal conductivity is calculated by using the unit cell volume. For 2D systems, you will need to multiply the factor (c/d) where c is the unit-cell length along the z-axis (perpendicular to the 2D sheet) and d is the thickness of the 2D sheet.

**Reply by apandit2021** [reply]
Dear Dr. Tadano,

Thank you very much for the information. 

Best,
Abhiyan

**Reply by apandit2021** [reply]
Dear Dr. Tadano,

In your above statement about lattice thermal conductivity (k_l), can you explain briefly how does multiplying the k_l value (obtained from ALAMODE) by the factor c/d gives the k_l of 2D system?

Best,
Abhiyan

**Reply by ttadano** [reply]
When calculating k_l, the summed value inside the first Brillouin zone is devided by the unit-cell volume. For 2D systems, we need to include a vacuum region in the unit cell, but ALAMODE still uses the unit-cell volume for computing k_l. Therefore, you will need to multiply (c/d) for obtaining a k_l value of a 2D material.

**Reply by apandit2021** [reply]
Dear Dr. Tadano,

Thank you very much for the information. 

Best,
Abhiyan
